ncat/ncat_main.c

```diff
@@ -652,6 +661,12 @@ int main(int argc, char *argv[])
         }
     }
 
+    /*  --use-delimiter can be used only for broadcasting purposes.
+        So checking for listen mode is necessary before moving forward. */
+    if(o.delimiter_used == 1 && o.listen != 1) {
+        bye("--use-delimiter option is binded with -l option.");
```

Comment:
This error message is hard to understand. Copy a different one and just change the option name. But our goal should be to use this any time we are reading stdin, which would include listen mode as well. Look into it, because that would be the next project we'd tackle if we get this one done.